client/src/modules/list/index.jsx

```diff
@@ -60,12 +61,18 @@ class List extends Component {
   componentDidMount() {
     this.setState({ pendingForDetails: true });
 
-    this.fetchData().finally(() => {
-      this.setState({ pendingForDetails: false });
-      this.handleBreadcrumbs();
-      this.handleSocketConnection();
-      this.receiveWSEvents();
-    });
+    this.fetchData()
+      .then(() => {
+        this.handleBreadcrumbs();
+        this.handleSocketConnection();
```

Comment:
remove socket connection from here as we have HOC component right now. 
and change hame to handleRoomConnection() cause you'll be only joining a room right now

Reply:
this old branch, there is no this HOC here, this place should be changed in this task EOC-283,